web/src/app/profile/approved-researcher/components/AgreementForm.tsx

```diff
@@ -27,21 +27,20 @@ export default function ApprovedResearcherForm(props: ApprovedResearcherFormProp
   };
 
   return (
-    <div className="form-container">
+    <div className="form form__wrapper">
       {!submitted && (
         <form onSubmit={handleSubmit}>
-          I agree
           <input
             type="checkbox"
             name="agreed"
             onChange={() => {
               setAgreed(!agreed);
             }}
             checked={agreed}
-            className="input-checkbox"
             required
-          />
-          <button type="submit" className="submit-button">
+          />{" "}
```

Comment:
?

Reply:
linter thing, I'm used to seeing it and I think it just makes things more readable?

Reply:
If you're referring to the `{" "}` I would remove them. If we end up with lots of them they start to pollute the code and look messy.